pkg/runtime/namespace/namespace.go

```diff
@@ -38,11 +38,18 @@ import (
 	"github.com/arana-db/arana/pkg/util/log"
 )
 
-var _namespaces sync.Map
+var (
+	_namespaces sync.Map
+	_tmpns      sync.Map
```

Comment:
what's the meaning of `_tmpns`? suggest make a meaningful name.

Reply:
forget remove it, will fix latter